pandas/core/indexes/base.py

```diff
@@ -5926,3 +5931,22 @@ def _maybe_asobject(dtype, klass, data, copy: bool, name: Label, **kwargs):
         return index.astype(object)
 
     return klass(data, dtype=dtype, copy=copy, name=name, **kwargs)
+
+
+def get_unanimous_names(*indexes):
```

Comment:
do we ever get more than two here?

Would `indexes` instead of `*indexes` work>?

Reply:
yeah, we use this in `union_indexes` which operates on a list of indices. but most uses are explicitly two, so I figured this would make the typical syntax easier.

Reply:
can you type the input and output